open_spiel/python/mfg/algorithms/distribution.py

```diff
@@ -19,6 +19,22 @@
 import pyspiel
 
 
+def type_from_states(states):
+  """Get node type of a list of states and assert they are the same."""
+  types = [state.get_type() for state in states]
+  for state_type in types:
+    assert types[0] == state_type, f'types: {types}'
+  return types[0]
+
+
+def is_terminal_from_states(states):
```

Comment:
We could actually either:
- get rid of this, and instead use type_from_states(states) == pyspiel.StateType.TERMINAL on line 71, or
- replace the body of this function with "return type_from_states(states) == pyspiel.StateType.TERMINAL"

Reply:
I chose open 2.